static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -197,6 +186,17 @@ $pillars: (
                 fill: map-get($palette, commentCount);
             }
         }
+
+        &:hover {
+            .fc-item__avatar {
+                background-color: darken(map-get($palette, cutoutBackground), 3%);
+            }
+
+            .fc-item__timestamp,
+            .fc-trail__count--commentcount {
+                background-color: $darkerCardBackground;
+            }
+        }
     }
```

Comment:
I just moved this `&:hover` block into the `.fc-item--type-comment` block at the bottom of this module as it's better to have all `.fc-item--type-comment` styles grouped together.